Notebooks/historical_vote_proportions.ipynb

```diff
@@ -0,0 +1,1205 @@
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## To start off we will connect to Seattle's data base.\n",
+    "seattle = \"cdp-seattle-staging-dbengvtn\"\n",
+    "fireo.connection(client=Client(\n",
+    "    project=seattle,\n",
+    "    credentials=AnonymousCredentials()\n",
+    "))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center> <h3> Fetch all historical votes from data base</h3> </center>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<cdp_backend.database.models.Vote at 0x7f904a56db10>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904a56dbd0>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98c50>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98e90>,\n",
+       " <cdp_backend.database.models.Vote at 0x7f904aa98f90>]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "############ Select all of the records from the Vote table ############\n",
+    "\n",
+    "## If you want to specify a certain amount of records, just use an argument within fetch() function.\n",
+    "seattle_voting_records = list(db_models.Vote.collection.fetch()) \n",
+    "\n",
+    "## Preview file\n",
+    "seattle_voting_records[0:5]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total Voting Records To Date:\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "3368"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## Function to calculate total voting records to date:\n",
+    "\n",
+    "def total_vote_records(list_of_objects):\n",
+    "    '''\n",
+    "    Takes a list of database objects and prints out the length.\n",
+    "    '''\n",
+    "    total_records = len(list_of_objects)\n",
+    "    print(\"Total Voting Records To Date:\")\n",
+    "    return(total_records)\n",
+    "\n",
+    "total_vote_records(seattle_voting_records)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "* These files actually just contain information about the voting records we are interested in.\n",
+    "* In order to work with the files we need to \"unpack\" them using this information we retrieved.\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Function to unpack voting records with data retrieved\n",
+    "\n",
+    "def unpack_vote_objects(db_models_objects):\n",
```

Comment:
Can you simply change this function name from `unpack_vote_objects` to `unpack_db_models`

Reply:
Learning to call things "models". 
Done